client/jetpack-connect/constants.js

```diff
@@ -4,6 +4,7 @@ export const CALYPSO_REDIRECTION_PAGE = '/posts/';
 export const IS_DOT_COM_GET_SEARCH = 'isDotComGetSearch';
 export const JETPACK_ADMIN_PATH = '/wp-admin/admin.php?page=jetpack';
 export const JETPACK_MINIMUM_WORDPRESS_VERSION = '4.7';
+export const JPC_PATH_BASE = '/jetpack/connect';
```

Comment:
Users clicking on Jetpack Free should be redirected to `/jetpack/connect` and not to `/jetpack/connect/install`. They will be redirected there eventually if necessary.